lldb/source/Target/Target.cpp

```diff
@@ -3109,7 +3104,10 @@ bool Target::RunStopHooks() {
       auto result = cur_hook_sp->HandleStop(exc_ctx, output_sp);
       switch (result) {
       case StopHook::StopHookResult::KeepStopped:
```

Comment:
Should the "auto continue" override only apply to `KeepStopped` or all potential `HandleStop() -> StopHookResult` return values, or a specific set?
@jimingham

Reply:
Only KeepStopped.  If any thread reports AlreadyContinued, then we aren't doing this negotiation anymore, someone called `continue` directly.  Threads reporting NoPreference shouldn't be consulted here.  And RequestContinue means that thread goes along with continuing.
GetAutoContinue is formally different, since that is a stop-hook that's asking to auto-continue regardless of what value it returns from the stop hook callback, but in practice is the same as RequestContinue.